src/Sylius/Bundle/TranslationBundle/GedmoHandler/TranslationSlugHandler.php

```diff
@@ -128,6 +128,8 @@ public function postSlugBuild(SluggableAdapter $ea, array &$config, $object, &$s
                 }
             }
         }
+
+        $slug = $this->deleteUnnecessaryParentSlug($slug);
```

Comment:
*Primo:* Unfortunately ``$slug`` is passed by reference, so it's in fact used somewhere else ;)
*Secundo:* The reason is simple - clean code principles. Let's assume we have that line in the code:
```php
$slug = preg_replace('/^' . preg_quote($this->parentSlug.$this->usedPathSeparator, '/') . '/', '', $slug);
```
Even if your fluent in regular expressions, it takes a to figure out what does this line do. I would probably need a comment to explain it. Seeing this:
```php
$slug = $this->deleteUnnecessaryParentSlug($slug);
```
we are 100% sure what the hell is going on. Just following Uncle Bob advices ;)